### DISCUSSION TITLE
Migrating from the openHab zigbee binding to zigbee2mqtt

### DISCUSSION THREAD

**Discussion by obones** [Koenkk/zigbee2mqtt#17157] [General]
Hello,

I have a setup here with a BV2010/10 coordinator that works quite well with the openHAB Zigbee binding.
However, that binging does not properly handle quite a few devices that I have tried and zigbee2mqtt appears to be the recommended way to overcome those limitations.
I have thus stopped the openHAB zigbee binding and setup zigbee2mqtt on the same linux computer to use the same coordinator.
I believe I have copied over the proper network elements:
| Setting | openHab | zigbee2mqtt |
|---|---|--|
| Pan ID | 12345 | 12345 |
| Extended PAN ID | ABCDEF0123456789 | [0xAB, 0xCD, 0xEF, 0x01, 0x23, 0x45, 0x67, 0x89] |
| Network Security Key | ABCDEF0123456789ABCDEF0123456789 | [0xAB, 0xCD, 0xEF, 0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF, 0x01, 0x23, 0x45, 0x67, 0x89] |

zigbee2mqtt starts up just right but when I go to the "devices" tab in the web interface, the list is empty, even after waiting a few minutes.
Using the same security elements, I thought that this would allow for easy "migration" to the new software, but it seems that I have missed something here.
As I can see the MAC address in the openHAB interface, I decided to add them as children of the "devices" element in the config file, but that did not change the situation either.

Pairing a new device works fine and it appears in the configuration file but I would like to avoid repairing all devices if it's at all possible.

Thanks a lot for your help.

**Comment by lux73** [comment]
afaik there is no migration from OpenHAB Zigbee Binding to zigbee2mqtt without re-pairing all Devices

**Reply by obones** [reply]
So simply porting over the security elements are not sufficient? That's a pity but if there is no other choice...

**Reply by sjorge** [reply]
I haven't tried, but I think if you enable device joining and trigger the device to send a message, z2m might attempt to interview it.

Or open joining and wait for a bit might also do it I guess. But never tried.
